skops/io/tests/test_persist.py

```diff
@@ -42,10 +42,10 @@
     StandardScaler,
 )
 from sklearn.utils import all_estimators, check_random_state
-from sklearn.utils._tags import _safe_tags
+from sklearn.utils._tags import get_tags
+from sklearn.utils._test_common.instance_generator import _construct_instances
```

Comment:
this only works in the new sklearn though. We'd need to have wrappers in `fixes.py` to support both.

Reply:
I will try to make everything pass and then work on the separation at the end, it might make more sense since there are more incompatibilities in the same file. we can leave this unresolved as a reminder.